<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAnalyse</class>
 <widget class="QDialog" name="FaceAnalyse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#FaceAnalyse
{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>225</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>簇数：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>质点初始规则：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_initCenter"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>距离计算规则：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_disCalculate"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_sample">
      <property name="text">
       <string>生成样本数据</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_kmeans">
      <property name="text">
       <string>kmeans分类</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_k"/>
    </item>
   </layout>
  </widget>
  <zorder>pushButton_sample</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEdit_k</zorder>
  <zorder>lineEdit_k</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
